### ISSUE TITLE
Get Dependabot to base changes on develop branch

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#665]
**TL;DR**: Get Dependabot to make PRs to the develop branch instead of master.

**Narrative**
As a developer, I want to merge dependency updates into develop before master to check everything works. I may also want to merge multiple changes into develop and test on staging before updating master/production.

**Acceptance Criteria**
- [x] New Dependabot PRs come to develop
- [x] CI/CD pipeline still works for automatic deployments to staging & production.

**Implementation Notes**
Dependabot only makes PRs to the default branch. We can change the default branch to develop on GitHub, but need to ensure that that doesn't affect the CI/CD pipeline (lookit-orchestrator).
